### ISSUE TITLE
AVT 1 - React for the Header Base w/SideNav UI Shell has DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3578]
## Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
DAP - July 2019 Ruleset

## Detailed Description
Run DAP on the Header Base w/SideNav UI Shell. The following 3 violations:
![HeaderSideNav-1](https://user-images.githubusercontent.com/21676914/62135733-5877b200-b2a8-11e9-9470-2b896dfc18f3.png)
![HeaderSideNav-2](https://user-images.githubusercontent.com/21676914/62135743-5b72a280-b2a8-11e9-92a9-79bb6dd112d6.png)

**Comment by joshblack** [comment]
It seems like the contrast violation refers to our skip to content widget:

<img width="205" alt="Screen Shot 2019-12-05 at 11 33 38 AM" src="https://user-images.githubusercontent.com/3901764/70259130-3361e700-1753-11ea-8783-18d7120a60cb.png">

This has the following contrast:

<img width="604" alt="Screen Shot 2019-12-05 at 11 34 10 AM" src="https://user-images.githubusercontent.com/3901764/70259143-38bf3180-1753-11ea-98ac-e19a0c13cc2d.png">

The second issue is related to the demo environment because storybook wraps our work in a `<div>` with `role="main"`
